client/cli/src/client_proxy.rs

```diff
@@ -321,7 +321,7 @@ impl ClientProxy {
                 ),
                 is_blocking,
             ),
-            None => self.mint_coins_with_faucet_service(&receiver, num_coins, is_blocking),
```

Comment:
Oops, I should have fixed this in the short addresses change. My bad

Reply:
no worries, we don't have proper test coverage for faucet service. It's easy to miss that. Speculation in a past was that it's a temporary thing. Sorry for bad experience